.gitignore

```diff
@@ -2,4 +2,5 @@
 out
 target
 classes
-*.bak
\ No newline at end of file
+*.bakbuild
```

Comment:
typo?

Reply:
What is bakbuild and what about the original {{*.bak}} entry ?

Reply:
ah, I didn't notice that. Yup, that's got to be a typo!